test/neighborhood_search.jl

```diff
@@ -50,14 +50,23 @@
                                                                            max_corner = periodic_boxes[i].max_corner,
                                                                            search_radius,
                                                                            periodicity = true)),
+                GridNeighborhoodSearch{NDIMS}(; search_radius, n_points,
+                                              periodic_box = periodic_boxes[i],
+                                              cell_list = FullGridCellList(;
+                                                                           min_corner = periodic_boxes[i].min_corner,
+                                                                           max_corner = periodic_boxes[i].max_corner,
+                                                                           search_radius,
+                                                                           backend = Vector{Vector{Int32}},
+                                                                           periodicity = true)),
```

Comment:
Can you move the **FullGridCellList** initialization to a variable so that we don't get all of these linebreaks?

Reply:
Doesn't make it any better:
```julia
            full_grid_default = FullGridCellList(;
                                                 min_corner = periodic_boxes[i].min_corner,
                                                 max_corner = periodic_boxes[i].max_corner,
                                                 search_radius, periodicity = true)

            full_grid_vector = FullGridCellList(;
                                                min_corner = periodic_boxes[i].min_corner,
                                                max_corner = periodic_boxes[i].max_corner,
                                                search_radius,
                                                backend = Vector{Vector{Int32}},
                                                periodicity = true)
```

Reply:
I made it a bit better now by extracting `min_corner` and `max_corner`.